network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -16,7 +16,7 @@
 {% endblock %}
 
 {% block published_date %}
-  <span class="tw-h6-heading tw-text-gray-40 tw-my-1"> {{ page.first_published_at|date:"DATE_FORMAT" }} </span>
+  <span class="tw-h6-heading tw-text-gray-40 tw-py-1 tw-mb-0"> {{ page.first_published_at|date:"DATE_FORMAT" }} </span>
```

Comment:
Sorry for not making this clear before. I meant this:

```suggestion
  <span class="tw-h6-heading tw-text-gray-40 tw-py-1 tw-mb-0">{{ page.first_published_at|date:"DATE_FORMAT" }}</span>
```

If this works.